exporters/otlp/test/otlp_http_exporter_test.cc

```diff
@@ -466,6 +467,7 @@ TEST_F(OtlpHttpExporterTestPeer, ExportJsonIntegrationTestSync)
 TEST_F(OtlpHttpExporterTestPeer, ExportJsonIntegrationTestAsync)
 {
   ExportJsonIntegrationTestAsync();
+  google::protobuf::ShutdownProtobufLibrary();
```

Comment:
Is protobuf library loaded explicitly?

Reply:
This call is needed to delete any global objects that were allocated by the Protocol Buffer: libraryhttps://developers.google.com/protocol-buffers/docs/cpptutorial

Reply:
Thanks for sharing the doc. It looks optional in the doc. Also wondering whether live global objects should be considered as leak.

Reply:
yes it is optional, here we need it to prevent the leak report though.
Live global objects should not be considered as leak.